
<launch>
    <param name="use_sim_time" value="true"/>

    <!--启动配置好的rviz-->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find lwio)/rviz/lwio.rviz" />

    <!-- Parameters -->
    <param name="ParamPath" value="$(find lwio)/config/kitti_velodyne_icp_timedIvox.yaml" />
    <!--  /livox/lidar "/rslidar_points" /points_raw   /left/velodyne_points /velodyne_points   /kitti/velo/pointcloud-->
    <param name="LidarTopic" value="/kitti/velo/pointcloud" />
    <!-- /imu/data     imu_raw-->
    <param name="ImuTopic" value="/imu/data" />
    <param name="GnssTopic" value="/fix" />  
    <param name="OdomTopic" value="/odom_scout" />
    <param name="LidarType" value="1" />   <!--0: livox, 1:velodyne--> 
    <param name="MaxRangeThresh" value="70" />
    <param name="MinRangeThresh" value="3" />

    <node name="advanced_lio_node" pkg="lwio" type="advanced_lio_node" output="screen">
    </node>
</launch>